fix: enabling npm artifacts repo process update (rhdevdocs-5035) (#2640)
codenvy/che-docs: modules/end-user-guide/partials/proc_restoring-node-extra-ca-certs-variable-value.adoc

[id="restoring-node-extra-ca-certs-variable-value"]

> = Restoring correct NODE_EXTRA_CA_CERTS variable value

Reviewer: ```suggestion
= Configuring NODE_EXTRA_CA_CERTS to use a certificate
```

> Use the command described below to restore the correct NODE_EXTRA_CA_CERTS variable value every time before dependencies are downloaded.

Reviewer: Seems a big chonky.

```suggestion
Use the command below to set NODE_EXTRA_CA_CERTS to point to where you have your SSL/TLS certificate. 
```

.Procedure

* Run the following command in the terminal:
+

> [source, yaml]

Reviewer: ```suggestion
[source, bash]
```

----

> `export NODE_EXTRA_CA_CERTS=/public-certs/nexus.cer` <1>

Reviewer: per https://stackoverflow.com/questions/23788564/npm-add-root-ca/56239954#56239954

```suggestion
`export NODE_EXTRA_CA_CERTS=/public-certs/nexus.cer` <1>

Then download dependencies with:
`npm install` or `yarn`
```
